### PR TITLE
Finess published package: Many browser testing, ES6 (babel), ES linting

### PR DESCRIPTION
**Pull request by juanca**
@naganowl @naomiajacobs

### CODE REVIEW

File: lib/actions/model.js [hunk 1]
```diff
@@ -39,24 +40,24 @@ module.exports = {
...
       if (trackKey) xhrs[trackKey] = xhr;
 
       return xhr;
     };
   },
 
-  validate(brainstemKey, modelId, attributes, options) {
-    /* should be extremely similar to fetch and save */
-  },
+  // validate(brainstemKey, modelId, attributes, options = {}) {
```
Comment (naganowl) [inline lib/actions/model.js]: Remove this or make a no-op?

File: spec/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  extends: "airbnb",
+  env: {
+    jasmine: true,
+  },
+  plugins: [
```
Comment (naganowl) [inline spec/.eslintrc.js]: Do we need these here and in the top level file? Perhaps we can import the top level one and extend that?

**Comment by juanca** [comment]
I need to make a quick change in a different PR and publish. Revisions for this PR will go in another PR. 

<sub>PR PR PR</sub>

**Comment by juanca** [comment]
Revisions @ https://github.com/mavenlink/brainstem-redux/pull/9

**Pull request merged**